tests/test_pythoncapi_compat.py

```diff
@@ -62,7 +62,7 @@ def build_ext():
     display_title("Build test extensions")
     if os.path.exists("build"):
         shutil.rmtree("build")
-    cmd = [sys.executable, "setup.py", "build"]
+    cmd = [sys.executable, "setup.py", "build", "--debug"]
```

Comment:
although runtests.py runs tests in a subprocess, I could debug my segfaults by running
```
gdb --args pypy3.11 tests/test_pythoncapi_compat.py -v
```

Using a debug build makes it easier